notebooks/advanced-feature-transformations-explain-local.ipynb

```diff
@@ -299,7 +300,7 @@
     "# LGBMExplainableModel can be replaced with LinearExplainableModel, SGDExplainableModel, or DecisionTreeExplainableModel\n",
     "# explainer = MimicExplainer(clf.steps[-1][1], \n",
     "#                            x_train, \n",
-    "#                            LGBMExplainableModel, \n",
+    "#                            LinearExplainableModel, \n",
```

Comment:
why using LinearExplainableModel?

Reply:
jupyter notebook autosave....
I'll fix it

Reply:
fixed